src/Template/Element/sql_log_panel.ctp

```diff
@@ -63,7 +63,7 @@ $noOutput = true;
             <tbody>
                 <?php foreach ($queries as $query): ?>
                 <tr>
-                    <td><?= h($query['query']) ?></td>
+                    <td><pre><?= SqlFormatter::format($query['query'], false) ?></pre></td>
```

Comment:
This should stil use `h()`, as entity encoding is only being done when using syntax highlighting.